charts/crossplane-leaf-install/Chart.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v2
+name: crossplane-leaf-install
+description: A Helm chart to install Crossplane roles and policies per leaf cluster definition
+
+type: application
+
+version: 0.0.1
+
+appVersion: "1.16.0"
+
+home: https://github.com/weaveworks/profiles-catalog
+
+keywords:
+- operator
+- crossplane
+
+annotations:
+
+maintainers:
+  - name: Weaveworks
+    email: support@weave.works
+
+dependencies:
+- name: crossplane
+  version: "1.7.0"
+  repository: "https://charts.crossplane.io/stable"
```

Comment:
This has to removed. 
```

dependencies:
- name: crossplane
  version: "1.7.0"
  repository: "https://charts.crossplane.io/stable"
```

Crossplane gets installed with no cloud provider and no cloud provider CRDs.

Then, [this](https://github.com/filecoin-project/gitops-root/blob/main/.weave-gitops/apps/crossplane/base/crossplane-provider/aws-provider.yaml) configuration is applied and Crossplane then install the AWS CRDs. So, Crossplane either needs to be seperated out or you can keep the current flow and just remove this section

Reply:
removed